lib/URI.pm

```diff
@@ -1085,6 +1116,34 @@ examples:
 This value can be set to ";" to have the query form C<key=value> pairs
 delimited by ";" instead of "&" which is the default.
 
+=back
+
+=head1 ENVIRONMENT VARIABLES
+
+=item URI_HAS_RESERVED_SQUARE_BRACKETS
+
+Before version 5.11, URI treated square brackets as reserved characters
+throughout the whole URI string. However, these brackets are reserved
+only within the autority/host part of the URI and nowhere else (RFC 3986).
+
+Starting with version 5.11, URI takes this distinction into account.
+Setting the environment variable c<URI_HAS_RESERVED_SQUARE_BRACKETS>
+(programmatically or via the shell), restores the old behavior.
+
+ Note: This environment variable is just an initialiser and has to be set
+       before module URI is used/required. Changing it at run time has no effect.
```

Comment:
Is this part of the code block in the POD? If so, it should have `#`. Otherwise, I would suggest to put some emphasis on the note bit.

Reply:
Will be fixed.